examples/Inference/basic_inference/basic_inference.py

```diff
@@ -0,0 +1,63 @@
+import filelock
+
+import pathlib
+
+import os
+
+import torch
+
+from torch.profiler import ProfilerActivity
+
+import torchvision as tv
+import torchvision.transforms as transforms
+
+from _torch_batch_process import (
+    TorchBatchProcessor,
+    torch_batch_process,
+)
+
+from model import get_model
+
+
+class MyProcessor(TorchBatchProcessor):
+    def __init__(self, init_info):
+        self.model = get_model()
+        self.device = init_info.default_device
+        self.model.eval()
+        self.model.to(self.device)
+        self.profiler = torch.profiler.profile(
+            activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA],
+            schedule=torch.profiler.schedule(wait=1, warmup=1, active=2, repeat=2),
+            on_trace_ready=torch.profiler.tensorboard_trace_handler(init_info.tensorboard_path),
+        )
+        self.worker_rank = init_info.rank
+
+    def process_batch(self, batch, batch_idx) -> None:
+        model_input = batch[0]
+        model_input = model_input.to(self.device)
```

Comment:
we provide a helper method to serialize various types of data to device for training, `context.to_device`, wonder if that might be useful for inference as well?